File: testcases/src/main/java/duplicates/Res_duplicate1.java
package duplicates;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
// TODO check thqat by completion it doesn't suggest "duplicates" two times  in clients/HTTP requests
@Path("duplicates")
public class Res_duplicate1 {

    @GET
    public String foo() { return "foo"; }

    @GET
    @Path("bar")
    public String bar() {
        return "bar";
    }
}

File: testcases_kt/src/main/kotlin/duplicates/Res_duplicate1.kt
package duplicates

import javax.ws.rs.GET
import javax.ws.rs.Path

// TODO check thqat by completion it doesn't suggest "duplicates" two times  in clients/HTTP requests
@Path("duplicates_kt")
class Res_duplicate1 {
    @GET
    fun foo(): String {
        return "foo"
    }

    @GET
    @Path("bar")
    fun bar(): String {
        return "bar"
    }
}
